src/app/pages/home/services/tasks.service.ts

```diff
@@ -93,8 +93,12 @@ export class TasksService {
    */
   getNextTask(tasks: Task[]): Task | undefined {
     if (tasks) {
-      return tasks.find(task => !this.isTaskExpired(task))
+      const nextTasksNow = tasks.filter(task => this.isTaskStartable(task))
+      if (nextTasksNow.length) {
+        return nextTasksNow.sort((a, b) => a.order - b.order)[0]
```

Comment:
if two tasks have the same order, will it show by scheduled time then?

Reply:
Yeah, the default is by scheduled time so it will not reorder if they are the same.